macos/FluentUI/Button.swift

```diff
@@ -193,6 +193,10 @@ open class Button: NSButton {
 
 	private var hoverBackgroundColor: NSColor {
 		if #available(macOS 10.14, *) {
+			if (fillColor == .clear)
+			{
```

Comment:
swift (and therefore fluentui-apple) convention is same-line brackets like the rest of this file uses (same comment below). We really should get swift-format hooked up to make this automatic (was really hoping @Saadnajmi would do that this week :))

Reply:
Also looks like there's an extra space between "return" and "NSColor" here and below

Reply:
Decided to add a clang format to the objective C code Amanda's team normally owns.

Tl;DR from my last FHL, it's easy to add a formatter as an optional thing, it's hard to include it as a dependency to this repo / enforce it is on every developers machine. I suppose at the very least we can add a swift-format file though...

Reply:
Also @anandrajeswaran I've got about 4 changes of varying size I'm doing this week so I highly doubt I'll get to add it :/

Reply:
if you run "swiftlint autocorrect" on the root of the repo iOS swift lint file works for mac code too. @chiuam  did you install swiftlint on your machine?